src/cmds/scripts/pbs_init.d.in

```diff
@@ -113,22 +113,21 @@ lc_host_name()
 #
 : check_started
 check_started() {
+	ps_out=`ps -p $1 -o args 2> /dev/null | tail -1`
```

Comment:
Thank you for formatting this file. One optional comment, can we also change `` to `$()` syntax?